addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -32,6 +34,16 @@ <H3>Spider Add-On</H3>
 Other add-ons which use Traditional Spider functionality have also been re-worked to support the Spider add-on, including: Quick Start, Form Handler, GraphQL, OpenAPI, SOAP, and the Automation Framework. 
 More details are given below.
 
+<H3>Import/Export Add-On</H3>
+
+The Import/Export add-on allows to import/export data (e.g. HTTP Messages, URLs) to/from ZAP, it supersedes the following add-ons which will no longer be available in the marketplace:
+<ul>
+  <li>Import files containing URLs
+  <li>Log File Importer
+  <li>Save Raw Message
+  <li>Save XML Message
+</ul>
```

Comment:
Should this also mention superseded core functionality or is that obvious enough and covered by the api part?

Reply:
Added, looks ok?